cmd/wego/main.go

```diff
@@ -16,18 +16,54 @@ import (
 	fluxBin "github.com/weaveworks/weave-gitops/pkg/flux"
 )
 
+var VERSION = "0.0.0-dev.0"
+
 var options struct {
 	verbose bool
 }
 
 var rootCmd = &cobra.Command{
-	Use:   "wego",
-	Short: "Weave GitOps",
+	Use:           "wego [subcommand]",
+	Version:       VERSION,
+	SilenceUsage:  true,
+	SilenceErrors: true,
+	Short:         "Weave GitOps",
+	Long:          "Command line utility for managing Kubernetes applications via GitOps.",
+	Example:`
+  # Get verbose output for any wego command
+  wego [command] -v, --verbose
+
+  # Get wego app help
+  wego help app
+
+  # Add gitops application from GitHub repository or local git repo
+  wego app add podinfo
+
+  # Get status of deployed application
+  wego app status podinfo
+
+  # Get help for wego app add command
+  wego app add -h
+  wego app help add
```

Comment:
The second form doesn't work; `wego help app add` does, though

Reply:
I get *something* close to help, but not quite correct:
```$ bin/wego app help add

  Add:    Add an application to wego control,
  Status: Get status of application under wego control

Usage:
   app [command]

Examples:

  # Add an application to wego from local git repository
  wego app add ./myapp --name myapp

  # Status an application under wego control
  wego app status myapp

Available Commands:
  add         Add a workload repository to a wego cluster
  list        List applications
  status      Get status of a workload under wego control

Flags:
  -h, --help   help for app

Global Flags:
      --namespace string   gitops runtime namespace (default "wego-system")
  -v, --verbose            Enable verbose output

Use " app [command] --help" for more information about a command.
```
Anyway, I've fixed the example, coming shortly.